R/functions.R

```diff
@@ -3,6 +3,34 @@
 #' Return id of the planning googlesheet
 gs_id <- function() "1HLtyGK_csi5W_v7XChxgTuVjS-RKXqc0Jxos1RBqpwk"
 
+
+#' Sanity checks for the google sheet.
+#' This function is used in data pipe-lines to check that everything
+#' in the google sheet is as expected.
+#' It will throw errors or warnings if something goes wrong.
+#'
+#' @details the function can be used in a dplyr pipe, which is why
+#' it requires and returns the `.data` argument.
+#' @param .data a data frame (or derivative) loaded from a google sheet
+sanity_checks <- function (.data) {
+  # https://github.com/hadley/assertthat/issues/41
+
+  # check if there are NA columns not supposed to be empty
+  required_columns <- c('Thema', 'Subthema', 'Start')
```

Comment:
Please use double quotes for strings when possible.

Reply:
I will, thank you!